prefect2/data_source_block_documents.go

```diff
@@ -0,0 +1,138 @@
+package prefect2
```

Comment:
We never published a `prefect` provider on the Terraform Registry, so I'd propose that we simply call this `prefect`

Reply:
@jawnsy for people using this automation for Prefect 1, wouldn't changing this cause problems?

Reply:
No, we're not planning to publish a Terraform provider for Cloud 1.0, so yes, I think just `prefect` makes sense!